graphstorm-processing/LICENSE

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
we already have an apache license. do we need another license?

Reply:
Resolving as @classicsong  made the same comment, let's discuss there.